firefoxThemes/options/option.css

```diff
@@ -11,42 +11,49 @@ html, body {
   background-color: var(--header-color);
   color: var(--foreground-color);
 }
+
 section {
   position: relative;
   margin-top: 2em;
 }
+
 h1 {
   text-align: center;
 }
+
 label {
   color: var(--foreground-color);
 }
+
 /*Custom Checkbox*/
 span {
-  position:relative;
-  top:5px;
+  position: relative;
+  top: 5px;
   display: inline-block;
   width: 12px;
   height: 12px;
-  border: 2pt solid #00000070;
+  border: 2pt solid var(--checkbox-border-color,#00000070);
   border-radius: 3px;
 }
 
-span:hover{
+span:hover {
   border: 2pt solid #666879;
 }
 
-span:active{
+span:active {
   background-color: #9999;
   border: 2pt solid #888888;
 }
-.checked{
-  background-color: var(--checkbox-color,red);
+
+.checked {
+  background-color: var(--checkbox-color, red);
 }
-.checked::after{
-  content: var(--checkmark,"✓");
-  position:relative;
-  bottom:5px;
-  font-weight:bold;
-  color:var(--checkmark-color,white);
+
+.checked::after {
+  content: "♥";
```

Comment:
Change the checkmark to instead be a heart icon. Naturally it changes color based on theme.

Reply:
Interesting! I like the idea,

Are the heart's off center on your machine?

![image](https://user-images.githubusercontent.com/15972415/132492348-70677ad8-be88-4941-b1f1-0efafc58cbe6.png)

Reply:
Created a possible fix for the out-of-bounds checkmark. Not sure if it will succeed as debugging with the responsive design mode doesn't help out much reproducing the issue.

Reply:
Just checked again, and it's still off center. Might just be a Linux thing. Again, not too concerned.